### PR TITLE
added octo-corp block

### PR DESCRIPTION
**Pull request by tuansydau**
Added octo-corp block

### CODE REVIEW

File: src/icons/OctocorpLogo.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      <defs>
+        <filter
+          id="filter0_d_452_4003"
+          x="0"
+          y="-0.000244141"
+          width="222"
+          height="222"
+          filterUnits="userSpaceOnUse"
+          color-interpolation-filters="sRGB"
+        >
```
Comment (smyzin) [inline src/icons/OctocorpLogo.tsx:70]: It's better to clean up SVG files and leave only what is needed (circle, path, etc.). Other stuff, like `drop-shadow` or `box-shadow`, can be added via tailwind.

File: src/icons/OctocorpLogo.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+          d="M141.408 57.4023C157.788 67.6264 168.694 85.8649 168.694 106.662C168.694 138.668 142.864 164.615 111.001 164.615C79.1387 164.615 53.309 138.668 53.309 106.662C53.309 85.8649 64.2147 67.6264 80.595 57.4023"
+          stroke="#FF7BED"
+          stroke-width="4"
+          stroke-linecap="round"
+          stroke-linejoin="round"
+        />
+        <path
+          d="M74.2364 74.0637C66.5985 82.7493 61.9634 94.1619 61.9634 106.662C61.9634 133.867 83.9186 155.922 111.002 155.922C138.085 155.922 160.04 133.867 160.04 106.662C160.04 94.1619 155.405 82.7493 147.767 74.0637"
+          stroke="#FF7BED"
+          stroke-width="4"
```
Comment (smyzin) [inline src/icons/OctocorpLogo.tsx:35]: You can add `stroke` & `strokeWidth` via tailwind classes. Use them instead

File: src/components/Octocorp.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  maxWidth,
+  space,
+} from 'classnames/tailwind'
+
+const textCard = classnames(space('space-y-2'), maxWidth('max-w-400'))
+const textCardHeader = space('space-y-4')
+const textCardBody = space('space-y-6')
+const wrapper = classnames(
+  display('flex'),
+  flexDirection('flex-col', 'lg:flex-row'),
```
Comment (smyzin) [inline src/components/Octocorp.tsx:18]: Maybe it's better to set `md:flex-row`?

File: src/components/Octocorp.tsx [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  alignItems('items-center'),
+  space('space-y-40'),
+  margin('mb-72')
+)
+
+export default function () {
```
Comment (smyzin) [inline src/components/Octocorp.tsx:24]: Did you check it on screen with `280px` size? Is everything okay?

**Review by smyzin — CHANGES_REQUESTED** [review]

File: src/components/Octocorp.tsx (hunk 4 above)
Comment (ohpaul) [reply src/components/Octocorp.tsx:24]: I don't think we want to use very big `margin` or `padding`. If everything can be handled by `space` would be optimal but using alternatives are fine as well such as `height`

**Review by ohpaul — APPROVED** [review]
lgtm :)

**Pull request merged**